xgbgen/main.go

```diff
@@ -49,16 +46,26 @@ func main() {
 	c := newContext()
 	c.Morph(xmlBytes)
 
+	outFile, err := os.Create(fmt.Sprintf("%s/%s.go", *packageName, *packageName))
+	if err != nil {
+		panic(err)
```

Comment:
`log.Fatal(err)` to be consistent with other uses.

Reply:
Fixed by cbed66aa3ac97f6bb2f59ad18df2472a2d4368bd